### PR TITLE
Hello world code review

### PR DESCRIPTION
**Pull request by Hidar37**
-add index.html
-added style.css
-added some markup in index.html
-added some style to page title (h1)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+test.md
+MIT.md
+app_screenshot.png
```
Comment (sja-thedude) [inline .gitignore:3]: - [ ] Kindly add node_modules in .gitignore as per the requirements.
- [ ] Kindly remove MIT.md from this file.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Hidar37, This is @sja-thedude your code reviewer.

## Welcome to Microverse!! 🎊 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ All linters passing
✔️ Correct Gitflow

## Required 
- [ ] You can watch the attached video to make changes [here](https://www.loom.com/share/e41329c7187645afa576eaafab5a9a53).
- [ ] Kindly make sure to link MIT.md to the license in the Readme file and also make changes in MIT.md by adding your name in place of name, year and also add the app type.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -71,5 +41,3 @@ Give a ⭐️ if you like this project!
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (tresorsawasawa) [inline README.md:43]: - [ ] As mentioned by the previous reviewer, kindly make sure the `[MIT]` license link is not broken to make your README professional. right now it is clicked it brings an error page. To fix that issue just choose one o the following solutions:
 - You can edit `[MIT](./LICENSE)` to this `[MIT](./MIT.md)` as the license file in your root directory is name `MIT.md` and not `LICENSE`.
 - Or just edit the `MIT.md` file in the root directory to `LICENSE` to match the path of the file/link

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @Hidar37 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smunozmo — APPROVED** [review]
Hi @Hidar37, 👋 

## Highlights 👏 

- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

# APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**